test/bucket_test.js

```diff
@@ -206,22 +209,21 @@ describe("Bucket", () => {
           safe: true,
         }).createCollection("foo", {headers: {Baz: "Qux"}});
 
-        sinon.assert.calledWithMatch(requests.createCollection, "foo", {
-          bucket: "blog",
-          headers: {Foo: "Bar", Baz: "Qux"},
-          safe: true,
-        });
+        sinon.assert.calledWithMatch(requests.createRequest, "/buckets/blog/collections/foo", {
+          data: {id: "foo"},
+          permissions: undefined
+        }, { headers: {Foo: "Bar", Baz: "Qux"}, safe: true });
```

Comment:
Maybe spread the options on several lines here and elsewhere for consistency and as it really improves legibility to me.